piman_README.md

```diff
@@ -87,7 +87,48 @@ The reinstallation process uses `/utility/power_cycle.py` to turn the pi at the
 network boot to install the pi.  
 To perform a network boot, the DHCP, TCP, and TFTP threads should be running.  
 
-  
+#### Getting mac address of rasberry pi(s)
+``` python
+def mapper(switch_address,interface, port):
+    for portNum in port:
+        power_cycle.power_cycle(switch_address,interface, portNum)
+    time.sleep(30)
+    mac_mapper.mac_mapper()
+```
+`mapper()` is a function inside the `piman.py` file. It ultimately encompasses two functions `power_cycle.power_cycle()` and `mac_mapper.mac_mapper()` which will power cycle the pis causing them to try PXE boot and afterwards getting the mac address of each specified pi. This output is then printed to console as well as `mac_mapper.txt`
+
+**How to use mapper()**
+
+The official implementation is specified as `mapper(switch_address,interface, port)`.
+
+`switch_address` (String) is your specified ip ending with .129.
+
+`interface` (Int) use `ifconfig` to figure out the interface that the pis are on
+
+`port` (int[]) is the specified remote pi that you want to access, ranging from either ports 1-10 or 11-20 depending on your team number.
```

Comment:
team is a class concept, not a general one, so you should just stick to the port number